umm-spec-lib/src/cmr/umm_spec/iso19115_2_util.clj

```diff
@@ -100,6 +100,9 @@
 (def extent-xpath
   "/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent[@id='boundingExtent']")
 
+(def extent-alt-xpath
+  "/gmi:MI_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent[@id='boundingExtent']/gmd:geographicElement/gmd:EX_GeographicDescription[@id='GranuleSpatialRepresentation']/gmd:geographicIdentifier/gmd:MD_Identifier")
```

Comment:
Use `str` to split this crazy long xpath onto multiple lines.

Reply:
Done.